recipes/refseq_masher/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "refseq_masher" %}
+{% set version = "0.1.0" %}
+{% set sha256sum = "3d4460966f5a7994b66c2c87cfe5a0116d8b2bdf6a9b8b0e4ea57cd891e26708" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256sum }}
+
+build:
+  skip: True # [py27]
+  skip: True # [not linux]
+  number: 0
+  script: $PYTHON setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - numpy
+        - pandas
+        - click
+        - mash
+test:
+    commands:
+        - refseq_masher --version
+        - refseq_masher --help
```

Comment:
Can you try:
```yaml
test:
  commands:
    - LANG=C.UTF-8 refseq_masher --version
    - LANG=C.UTF-8 refseq_masher --help
```
instead of skipping the macOS build?

Reply:
Thanks for the tip! I was wondering how to get it working on the MacOS build. Fixed in 898b3bd8b015425cdf81b3e5e6d921e302cf06ad

Reply:
It's actually a new thing that the macOS builds don't have that locale set -- previously it was (more or less) the other way around, hence the need for the `extended-base` image.
The macOS change must've come with the Travis CI -> CircleCI change. I think we'll fix that soon so that those `LANG=...` workaround aren't needed -- but your PR should have to wait for that 😉